packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountGroup.tsx

```diff
@@ -108,27 +89,45 @@ export const AccountGroup = ({
     const heading = getGroupLabel(type, hideLabel);
 
     return (
-        <div>
+        <Column gap={spacings.xxxs}>
             {heading !== null && (
-                <Header
-                    $isOpen={isOpen}
-                    onClick={!keepOpen ? onClick : undefined}
-                    data-testid={`@account-menu/${type}`}
+                <Box
+                    position={{ type: 'sticky', top: 0 }}
+                    zIndex={30}
+                    cursor={!keepOpen ? 'pointer' : 'default'}
```

Comment:
`"default"` or `undefined`?

Reply:
I guess it's the same.

Reply:
Yes and no, with undefined it's possible to change cursor behaviour from top level component. Just thinking if we actually want to use default if we don't want to change cursor behaviour.